torch/_export/passes/lift_constants_pass.py

```diff
@@ -148,18 +148,18 @@ def lift_constants_pass(
                 constant_kind = InputKind.CUSTOM_OBJ
                 constant_fqn = constant_attrs.get(constant_val)
                 if constant_fqn is not None:
-                    _, _, constant_name = constant_fqn.rpartition(".")
+                    constant_name = constant_fqn.replace(".", "_")
                 else:
-                    constant_name = f"_lifted_custom_obj{num_custom_obj}"
+                    constant_name = f"lifted_custom_obj{num_custom_obj}"
```

Comment:
I wonder if it's better to just change these names to the same prefixes as below, like `f"obj_{num_custom_obj}"` and `f"c_{num_tensor_constants}"`? Because to a user I don't think they would care if something was lifted during tracing in AOTAutograd or lifted later in this pass.

Reply:
Ah, the `run_placeholder_naming_pass()` after this should take care of the prefixing! But I do like those names instead, they are more concise.